javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentInitializer.java

```diff
@@ -246,57 +143,11 @@ private static boolean isDebugMode() {
     return false;
   }
 
-  /**
-   * Search for java or agent-tracer sysprops which indicate that a custom log manager will be used.
-   * Also search for any app classes known to set a custom log manager.
-   *
-   * @return true if we detect a custom log manager being used.
-   */
-  private static boolean isAppUsingCustomLogManager() {
-    String tracerCustomLogManSysprop = "otel.app.customlogmanager";
-    String customLogManagerProp = System.getProperty(tracerCustomLogManSysprop);
-    String customLogManagerEnv =
-        System.getenv(tracerCustomLogManSysprop.replace('.', '_').toUpperCase());
-
-    if (customLogManagerProp != null || customLogManagerEnv != null) {
-      log.debug("Prop - customlogmanager: " + customLogManagerProp);
-      log.debug("Env - customlogmanager: " + customLogManagerEnv);
-      // Allow setting to skip these automatic checks:
-      return Boolean.parseBoolean(customLogManagerProp)
-          || Boolean.parseBoolean(customLogManagerEnv);
-    }
-
-    String jbossHome = System.getenv("JBOSS_HOME");
-    if (jbossHome != null) {
-      log.debug("Env - jboss: " + jbossHome);
-      // JBoss/Wildfly is known to set a custom log manager after startup.
-      // Originally we were checking for the presence of a jboss class,
-      // but it seems some non-jboss applications have jboss classes on the classpath.
-      // This would cause jmxfetch initialization to be delayed indefinitely.
-      // Checking for an environment variable required by jboss instead.
-      return true;
-    }
-
-    String logManagerProp = System.getProperty("java.util.logging.manager");
-    if (logManagerProp != null) {
-      boolean onSysClasspath =
-          ClassLoader.getSystemResource(logManagerProp.replaceAll("\\.", "/") + ".class") != null;
-      log.debug("Prop - logging.manager: " + logManagerProp);
-      log.debug("logging.manager on system classpath: " + onSysClasspath);
-      // Some applications set java.util.logging.manager but never actually initialize the logger.
-      // Check to see if the configured manager is on the system classpath.
-      // If so, it should be safe to initialize jmxfetch which will setup the log manager.
-      return !onSysClasspath;
-    }
-
-    return false;
-  }
-
   private static boolean isJavaBefore9() {
     return System.getProperty("java.version").startsWith("1.");
   }
 
-  private static boolean isJavaBefore9WithJfr() {
+  public static boolean isJavaBefore9WithJfr() {
```

Comment:
can move this to AgentInstaller also

Reply:
the method uses 
```
Thread.currentThread().getContextClassLoader().getResource(jfrClassResourceName) != null
```

which I guess might be different in the AgentInstaller (agent classloader). Should I change it to the bootstrap claassloader?

Reply:
`Thread.currentThread().getContextClassLoader()` won't be different since it's based on the current thread and not the current class you are in

Reply:
done